__tests__/app.test.js

```diff
@@ -189,3 +189,78 @@ describe('Post request /api/articles/:article_id/comments',() => {
   })
   })
 })
+
+describe('Patch request /api/articles/:article_id', () => {
```

Comment:
Would also be worth a few more tests here, consider:
- What if inc votes was missing?
- What if inc votes had the wrong type?